include/crow/app.h

```diff
@@ -156,8 +156,9 @@ namespace crow
         /// Run the server on multiple threads using a specific number
         self_t& concurrency(std::uint16_t concurrency)
         {
-            if (concurrency < 1)
-                concurrency = 1;
+            if (concurrency < 2) // using 2 instead of 1 because 1 will be subtracted later.
```

Comment:
Move `2` to `const(expr)`.

Reply:
is this still necessary? if so, I'm not sure what you mean by it

Reply:
Something like `constexpr uint8_t MIN_THREAD_NUMBER = 2;`

Reply:
would it be necessary though? The 2 thread minimum is part of the framework's design.